canary/webrtc-c/jobs/runner.groovy

```diff
@@ -12,14 +12,12 @@ CREDENTIALS = [
     ]
 ]
 
-def buildProject(useMbedTLS, thing_prefix) {
-    echo 'Flag set to ' + useMbedTLS
+def buildWebRTCProject(useMbedTLS, thing_prefix) {
     checkout([$class: 'GitSCM', branches: [[name: params.GIT_HASH ]],
               userRemoteConfigs: [[url: params.GIT_URL]]])
 
-    def configureCmd = "cmake .. -DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_FLAGS_DEBUG=\"-g -O0\" -DCMAKE_INSTALL_PREFIX=\"\$PWD\""
```

Comment:
Hmm. I see some changes from main branch have got reverted? Can we confirm that this branch is rebased with master?